plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/configure/ui/CloudConnectorInitialConfigurationDialog.java

```diff
@@ -431,54 +427,26 @@ private void updateOKButtonStatus(boolean status) {
 	
 	@Override
 	protected void okPressed() {
-        IProject activeProject = EditorUtils.getActiveProject();
-        
-        boolean isIntegrationProject = false; 
-       	
-        try {
-			if(Arrays.asList(activeProject.getDescription().getNatureIds()).contains("org.wso2.developerstudio.eclipse.integration.project.nature")){
-				isIntegrationProject = true;	
-			}
-		} catch (CoreException e) {
-			log.error("Error occured when getting project nature", e);
+
+		IContainer location = EditorUtils.getActiveProject().getFolder(
```

Comment:
Shall we format this section?

Reply:
Formatted using the wso2 formatter in eclipse. It seems the way it is displayed in eclipse and in github is quite different.